src/Documentation/doc/advanced/bitnode_recommendation.md

```diff
@@ -0,0 +1,240 @@
+
+This BitNode's multipliers are harsh, but its rewards are mediocre, especially when comparing them to rewards in other BitNodes.
+
+## BitNode 12
+
+This is a special BitNode. In other BitNodes, the BitNode's multipliers are always the same, regardless of how many times you enter it, and the rewards only scale up to Source-File level 3. In BitNode 12, the BitNode's multipliers and the rewards scale to the Source File level, and the Source File level does not have a theoretical upper limit. For example, if you complete BitNode 12 one hundred times, you will have Source-File 12.100, and both the BitNode's multipliers and the rewards scale up to that level.
+
+Source-File 12 grants free NFG levels based on the Source-File's level.
+
+In this BitNode, you need to constantly change your strategy, depending on the current BitNode's multipliers. These multipliers become increasingly harsher when you go higher. It's recommended to unlock all mechanics before trying high levels.
+
+Note that, although there is no limit in theory, there is still a practical limit to how high you can go. You won't need to worry about that limit for a long time, though.
+
+## BitNode 13
+
+This BitNode unlocks [Stanek's Gift](./stanek.md). Stanek's Gift is a grid that you can freely put fragments on. There are many types of fragments, and each type boosts a different mechanic.
+
+This mechanic is versatile. It can boost many different mechanics. You can freely decide how to boost them by choosing appropriate fragments and arranging those fragments on the grid as you like.
+
+Stanek's Gift decreases many multipliers by 10%. "Church of the Machine God" faction offers 2 augmentations to gradually remove this penalty.
+
+You must accept Stanek's Gift before purchasing any augmentations. NFG is the only exception. Due to the effect of Source-File 7.3, you must accept Stanek's Gift before joining the Bladeburner division if you have that Source-File.
+
+This BitNode's multipliers are extremely harsh. You should prepare carefully before entering it.
+
+## BitNode 14
+
+This BitNode's Source-File buffs IPvGO's benefits and unlocks the ability to use cheat APIs. Cheat APIs may help you tremendously if you use them strategically, especially when playing against hard opponents. The Source-File's rewards:
+
+- Higher stat multipliers from Node Power.
+- Unlock `ns.go.cheat` APIs.
+- Higher maximum favor that you can gain for each faction from IPvGO.
+
+The last reward is pretty interesting. If you can consistently win several games in a row and get higher favor bonus, you will cut down the time to reach 150 favor points with IPvGO factions and have access to their augmentations much sooner.
+
+This BitNode's multipliers are fairly harsh, but IPvGO's multipliers are buffed significantly. If you use IPvGO, even a slightly improved version of the tutorial script will help you complete this BitNode without much trouble.
+
+# Order advice
```

Comment:
This is what people come here for, probably. It should appear above the deeper dives on the bitnodes IMO